docs/hub-deployment-guide/cloud-accounts/new-aws-account.md

```diff
@@ -84,7 +80,8 @@ of these are all 'Total CPUs' and hence larger nodes consume more quota.
   High Memory instances`) is requested specifically to be able to use GPU
   powered machines or machines with high amounts of RAM memory.
 
-  Our custom tailored hubs will require an increase in this quota.
+  By default, AWS grants us 640 quota here for GPU instances and 448 for
```

Comment:
640? Is that a typo that should be 64? I recall seeing this to be 64, but i may very well be mistaken

Reply:
haha, it is a typo! The other two we do get 640. I'll fix.